src/data/image.ts

```diff
@@ -19,8 +27,31 @@ const drive = google.drive({
   }),
 });
 
+const s3 = new S3({
+  accessKeyId: process.env.ACCESS_KEY_ID,
+  secretAccessKey: process.env.SECRET_ACCESS_KEY,
+  region: process.env.REGION,
+});
+
+/**
+ * Returns a list of all images stored in the currently configured
+ * S3 bucket. This method is cached for an hour and can be cleared by
+ * calling `memoize.clear(listS3Images)`.
+ */
+export const listS3Images = memoize(
```

Comment:
TODO: Clear this image cache if images were uploaded or deleted after a sync. Don't want to do it between every user image upload but at the end.... somehow.